src/main/java/ch/njol/skript/expressions/ExprRawString.java

```diff
@@ -46,6 +45,8 @@
 @Since("2.7")
 public class ExprRawString extends SimpleExpression<String> {
 
+	private static final Pattern HEX_PATTERN = Pattern.compile("&x((?:&\\p{XDigit}){6})");
```

Comment:
You can use Utils's hex pattern field, just make it public
https://github.com/SkriptLang/Skript/blob/aba2f87adde20d2ca2fbafc2d0eefc90faafd12a/src/main/java/ch/njol/skript/util/Utils.java#L643

Reply:
That's not the pattern I'm looking for. The pattern you mentioned matches `#abcdef` and the pattern I'm trying to match is `&x&a&b&c&d&e&f` (The spigot way of representing a hex code)

Reply:
Okay. Maybe add it to Utils as well like `HEX_PATTERN_SPIGOT`